requests_toolbelt/utils/dump.py

```diff
@@ -3,19 +3,116 @@
 
 from requests import compat
 
-
-__all__ = ('dump_response', 'dump_all')
+__all__ = ('dump_response', 'dump_all', 'Sanitizer', 'HeaderSanitizer')
 
 HTTP_VERSIONS = {
     9: b'0.9',
     10: b'1.0',
     11: b'1.1',
 }
 
+#: List of sensitive headers copied from
+#: https://github.com/google/har-sanitizer
+SENSITIVE_HEADERS = {
```

Comment:
I do not remember agreeing to wholesale import a list of headers to automagically sanitize. I thought we had agreed to merely allow users to provide a sanitizer, not give them a built-in one.

Reply:
My suggestion was to provide a method for providing your own, and one implementation that will do the right thing for most people. I imagine most people who need this will need to support this exact use-case: Redact headers commonly used to send sensitive data.

To me, the value of the feature increases tenfold if this implementation is provided, instead of every user of toolbelt having to implement this on their own in every project where this is needed.

Of course, this is your library, if you don't want it I can remove it.

Reply:
My concern is that this library is basically on life-support. Meaning that as this list grows, who will maintain it? And if people think this list is comprehensive, they will likely leak sensitive data without meaning to because they think this library is maintaining the list themselves.

Alternatively, what if we do the following:

1. Update `HeaderSanitizer` to require the headers set  be provided by the user.
1. Document the `SENSITIVE_HEADERS` list more thoroughly as a point-in-time snapshot from the repository with a Last-Updated date of today or whatever.
1. Indicate that if there are updates to the list that weren't made here users should use that themselves and/or send a PR to update this list.